helm/release.sh

```diff
@@ -1,7 +1,25 @@
 #!/bin/bash
-set -e
+
+set -eauo pipefail
+
+GIT_TAG=$(git describe --abbrev=0 --tags)
+
+TGZ_FILE=(ingress-azure-$GIT_TAG.tgz)
+
+if [ -f $TGZ_FILE ]; then
+  echo "File $TGZ_FILE already exists!"
+  exit 0
+fi
+
 echo " - running helm package"
-helm package ingress-azure --version 0.4.0
+helm package ingress-azure --version "$GIT_TAG"
+
 echo " - updating helm repo index"
 helm repo index . --url https://azure.github.io/application-gateway-kubernetes-ingress/helm
-echo " - done!"
\ No newline at end of file
+
+echo " - update helm templates"
```

Comment:
We should generate the Chart/Values before calling `helm package`.

Reply:
then you don't have to use `./update-index.py`

Reply:
Thanks!